pages/IssuePage.js

```diff
@@ -0,0 +1,22 @@
+const I = actor();
+
+module.exports = {
+
+  fields: {
+    familyManReference: '#D8caseReference',
+    placeOfMarriage: '#D8MarriagePlaceOfMarriage',
+    petitionersFullName: '#D8MarriagePetitionerName',
+    submit: 'button[type="submit"]'
+  },
+
+  fillFormAndSubmit() {
+    I.waitForElement(this.fields.familyManReference, 20);
+    I.fillField(this.fields.familyManReference, 'EZ11D81267');
+    // I.seeInField('petitionsFullName', 'Gary Ford');
```

Comment:
Do we still need this?

Reply:
I'll remove the code. We dont need it anymore

Reply:
Commented line removed. 👍